source/docs/tuner/tuner-swerve/swerve-system-requirements.rst

```diff
@@ -0,0 +1,75 @@
+Swerve Requirements
+===================
+
+The swerve project creator and :doc:`swerve API </docs/tuner/tuner-swerve/index>` have several limitations. These limitations are in-place to improve maintainability and debugging.
+
+* Only Phoenix 6 supported hardware (e.g. TalonFX, CANcoder, Pigeon 2.0).
+
+  * 8 TalonFX (4 steer, 4 drive)
+  * 4 CANcoder (4 steer)
+  * 1 Pigeon 2.0
+
+* Requires Phoenix 6 software released for year 2024 or newer.
+
+  * Firmware version should begin with 24.X
+  * Ensure Tuner shows server version as "2024" or newer
+
+* Temporary diagnostic server (or an existing 2024 robot project) must be running and Tuner should be connected to the robot.
+
+  * This allows the generator to perform it's setup and auto-calibration routines
+
+Some of these requirements are inforced via a mechanism called "precheck". You can see and refresh the precheck status by clicking the :guilabel:`Refresh` button in the top right.
+
```

Comment:
Maybe include a "recommended" section for CANivore use, Pro license, and the device renaming.

Reply:
I think the recommendations in the requirements checklist will suffice. I'll add licensing to the recommended requirements there.